app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 34
```

Comment:
As far as I'm informed, this property is deprecated and must be replaced to the `compileSdk 34`